openid-sharedsignals-framework-1_0.md

```diff
@@ -1461,6 +1522,37 @@ Cache-Control: no-store
 ~~~
 {: title="Example: Check Stream Status Response" #figstatusresp}
 
+The following is a non-normative example request to check an event stream's
```

Comment:
We intentionally got rid of subject-related status in #88 . Let's not add it back in here.